app/views/bz/_dynamic_syllabus.html.erb

```diff
@@ -132,15 +132,18 @@
           magic[:text] = "#{text} #{disp_name}"
           magic[:link] = link
           magic[:due_at] = due_at(obj, @current_user)
-          magic[:due_at_date] = overridden.due_at
+          # onboarding should always be first, so we pretend it is always due right now to make it
+          # at the top of the todo list
+          magic[:sort_order] = (disp_name == "Onboard to Braven") ? DateTime.now : overridden.due_at
```

Comment:
ugh, i don't like this, but whatever i'll merge. ideally it would go off the order in the modules list instead of a hardcoded name. i know we're going to change this name and then this code will be a NOOP and just add more noise...